docs/application/dotnet/samples/ui/overview.md

```diff
@@ -189,56 +189,56 @@ for TD:
 		<tr>
 			<td><img alt="" height="180" src="media/w46_animationsample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) AnimationSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/AnimationsSample" target="_blank"><strong>(W) AnimationSample (NUI)</strong></a></p>
 			<p>This sample application demonstrates how to animate NUI objects using the NUI animation.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w47imagesample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) ImageSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/ImageSample" target="_blank"><strong>(W) ImageSample (NUI)</strong></a></p>
 			<p>This sample application demonstrates how to load and control images.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w48scriptlayoutsample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) ScriptLayoutSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/ScriptLayoutSample" target="_blank"><strong>(W) ScriptLayoutSample (NUI)</strong></a></p>
```

Comment:
This comment is not about tizen-docs.

It looks like the title of readme.md of https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/ScriptLayoutSample is "ImageSample", not "ScriptLayoutSample".

![image](https://user-images.githubusercontent.com/9958374/55123278-3d076980-5145-11e9-8474-c036f21c8297.png)

Reply:
Fixed the title.
https://github.com/Samsung/Tizen-CSharp-Samples/blob/master/Wearable/ScriptLayoutSample